### ISSUE TITLE
Changes in the config file aren't applied

### ISSUE THREAD

**Issue by Gorgull** [Codes4Fun/RBDOOM-3-BFG#33]
I'd like to tweak a few options of the VR mod but my changes in the RBDOOM config file that's located in ~/Game Saves/... aren't taken into account and even reverted when quitting the game.

How does it work? 
What file should I edit?

**Comment by Codes4Fun** [comment]
Make sure the file you are editing is "Saved Games\id Software\RBDOOM 3 BFG\base\D3BFGVRConfig.cfg"
If you run another version of RBDOOM you could have another config file without VR in its name.

What are the options that you are trying to modify? The ones that I created should save, but there are many other options that are only temporary and are replaced by Doom3's data, so it would help to know the exact ones you are trying to modify. Also note, doom will rearrange the file, so if you add them to the bottom of the file, doom will change the order.

If for some reason they are not saving when modified in the file, try loading a game and modifying them in the console, they should save there.

**Comment by Codes4Fun** [comment]
~~Note, this is kind of dangerous, I found myself unable to play any normal doom levels after adding the new map data, until I cleaned by base directory of the map and a generated folder.~~

sorry that was intended for another issue

**Comment by Codes4Fun** [comment]
So investigating this issue further it seems that if you want to modify the cfg file, you have to also delete the base/savegame/profile.bin file, because that file also stores those variables and overrides what ever is in the cfg file.

I tested deleting the file and was able to load my saved games, so I don't think there is any issue there, but maybe back up the file just in case.

**Comment by Gorgull** [comment]
Removing the profile.bin everytime I modify D3BFGVRConfig.cfg works!

I'm actually trying to find the best locomotion parameters for me, and I'm modifying vr_moveMode, vr_strafing and vr_playerHeightCM

BTW I'm wondering why the first two are listed in CONSOLE.txt I found in your build, but not in the README.txt in your repo?

About the locomotion, could you add a move mode that requires holding click to walk (releasing the click stops the walk, even though the thumb still touches the pad)? 

Also, is there a way to change commands: 
- using the right-controller grip button instead of the touchpad to reload
- swap the light-trigger and the jump commands on the left-controller

Anyways, thanks a million for your answer and this crazy mod ;)

**Comment by Codes4Fun** [comment]
> BTW I'm wondering why the first two are listed in CONSOLE.txt I found in your build, but not in the README.txt in your repo?

The readme is originally from RBDoom 3 (the mod I am forked from), I've only updated it partly, mostly because this is all Alpha and I didn't want to maintain a bigger readme file until I am much further along. For example I am probably going to go through another bigger control change in the future.

> About the locomotion, could you add a move mode that requires holding click to walk (releasing the click stops the walk, even though the thumb still touches the pad)?

That should be mode 6 "hold walk". In my terminology there is "touch" which sounds exactly what it means, and "hold" which means you have to press and hold, and a "click" means to press and release.

> using the right-controller grip button instead of the touchpad to reload

They both allow you to reload (or they should unless there is some issue you are experiencing).

> swap the light-trigger and the jump commands on the left-controller

That is something I've been wanting to try.

**Comment by Gorgull** [comment]
Hold walk sounds good! Does mode 6 already exist (but not documented)?
Speed could be determined by the vertical position of the thumb on the touchpad (touching the edge of it = running for example)

I'll try the grip button to reload again tomorrow. 

By the way I really like the way the different touch pad functions are handled in Google Earth VR, with haptic feedback when the thumb navigates in between the different functions. It feels very natural to the point my mind almost imagines physical buttons...

Using the trigger for the flashlight feels a lot more intuitive to me, while jumping seems like much less frequently used as far as I know. 

As for crouching, have you considered detecting a physical jump (or at least a jumping "gesture") in parallel to a button press? Or maybe it actually works?

Thanks for answering so fast, I really appreciate it.

**Comment by Gorgull** [comment]
Sorry I just saw move mode 6 already exists in your latest build.
Can't wait to try it tomorrow back to my office!
